src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureEnvironment.cs

```diff
@@ -126,6 +126,8 @@ internal override IEnumerable<FieldSymbol> GetFieldsToEmit()
 
         public override Symbol ContainingSymbol => _topLevelMethod.ContainingSymbol;
 
+        public override bool AreLocalsZeroed => true;
```

Comment:
>public override bool AreLocalsZeroed => true; [](start = 8, length = 45)

Why is this the right implementation?

Reply:
Closures in the same method share the same `SynthesizedClosureEnvironment`. I believe we should keep the default behavior (return true), specially because two closures in the same method might have different AreLocalsZeroed flags. I think that's not possible right now, but it might become a concern when we enable attributes in local functions.